Assets/Scripts/PlayerMovement.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    // Base user speed
+    [SerializeField] private float speed = 5.0f;
+    private Rigidbody rb;
+
+    public int MaxJump = 1;
+    int JumpCount = 0;
+
+    // Start is called before the first frame update
```

Comment:
Can we get rid of these comments? We're going to see them a lot, no need to keep them in.

Reply:
Fine...

Reply:
Cool, will be happy to approve, merge and close down this PR when you've done it.

Reply:
DONE